src/lib/common/JsonHelper.h

```diff
@@ -31,6 +31,8 @@
 #include <vector>
 #include <map>
 
+
+// FIXME P10: should be renamed to JsonObjectHelper
```

Comment:
Two additional actions could be done:

1. Rename JsonHelper class to JsonObjectHelper class (I think this could be done in this same PR)
2. Split JsonHelper.h.cpp into JsonObjectHelper.h/cpp and JsonVectorHelper.h/cpp. I think this case is better to have them in only , as both classes are highly related and use some common logic such as toJsonString() function (which would force to have an additional couple jsonHelperCommon.h/cpp in the second case) and we have used this pattern in other places (e.g. Subscription related classes are all them in the same cpp/h module). However, I don't have an strong opinion on it so if you prefer the other way for me is ok and I can also implement it in this PR.

Reply:
I can try with something as you propose in order to see how looks like. Probably the level of reutilizanton will not be too high (as the way to add a key-value, for object, is different for the way of adding just the value, for vector) but let's see. Let's also see how it looks like from the point of view of the "user code".

If at the end we go with just one class them, two comments:

1. Maybe the name JsonConteiner first better than JsonHelper, as the later is less precise ("helper" may mean many things :), "container" is more precise).
2. I think is more clear to use a meanifull type (a enum) instead of a bool to select the container type:

```
JsonConatiner jco(jc::object);
...
JsonContainer jcv(jc::vector);
```

(alternative usggestion for jh:: namespace are welcome)

Reply:
At the end we will keep with two classes as it is now (discussed at skype).

The JsonHelper -> JsonObjectHelper renaming is pending.

Reply:
Rename done in 33b9f6de10f03a2f775bae250b4a3bcc66670eb6